index.js

```diff
@@ -33,8 +32,8 @@ const config = {
 const template = `
 <% if (jira.releaseVersions && jira.releaseVersions.length) {  %>
 Release version: <%= jira.releaseVersions[0].name -%>
-<% jira.releaseVersions.forEach((release) => { %>
-  * <%= release.projectKey %>: <%= jira.baseUrl + '/projects/' + release.projectKey + '/versions/' + release.id -%>
+<% jira.releaseVersions.sort((a, b) => a.projectKey.localeCompare(b.projectKey)).forEach((release) => { %>
+  * [<%= release.projectKey %>](<%= jira.baseUrl + '/projects/' + release.projectKey + '/versions/' + release.id %>)
```

Comment:
Not sure why but this outputs a new line after each entry. I haven't used this templating lib, so maybe you could advise @connorsmallman ?

Reply:
```suggestion
  * [<%= release.projectKey %>](<%= jira.baseUrl + '/projects/' + release.projectKey + '/versions/' + release.id -%>)
```
Try changing the closing tag to `-%>`

Reply:
I had it like that before, and it was the same.

Reply:
Adding `<% -%>` to the end of line solved it. Not sure if that's the best way or not.